lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -275,6 +275,35 @@ def test_discussion_url(self):
         self.assertIn('/api/discussion/v1/courses/{}'.format(self.course.id), response_discussion_url)
 
 
+    def test_org_query(self):
+        self.login()
+
+        # Create list of courses with various organizations
+        courses = [
+            CourseFactory.create(org='edX', mobile_available=True),
+            CourseFactory.create(org='edX', mobile_available=True),
+            CourseFactory.create(org='edX', mobile_available=True, visible_to_staff_only=True),
+            CourseFactory.create(org='Proversity.org', mobile_available=True),
+            CourseFactory.create(org='MITx', mobile_available=True),
+            CourseFactory.create(org='HarvardX', mobile_available=True),
+        ]
+
+        # Enroll in all the courses
+        for course in courses:
+            self.enroll(course.id)
+
+        response = self.api_response(qargs={'org':'edX'})
+
+        # Verify only edX courses are returned
+        self.assertEqual(len(response.data), 3)
+        for course_index in range(3):
+            result = response.data[course_index]['course']
+            self.assertEqual(result['org'], 'edX')
+
+        # Verify most recently enrolled course is staff only but still returned
+        self.assertFalse(response.data[0]['course']['courseware_access']['has_access'])
```

Comment:
This should be a separate test, if that test doesn't exist already.  It is unrelated to org querying.  If it does exist, this is unnecessary duplication.